x-pack/plugins/lens/public/xy_visualization/fitting_functions.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Fit } from '@elastic/charts';
+import { i18n } from '@kbn/i18n';
+
+export type FittingFunction = typeof fittingFunctionDefinitions[number]['id'];
+
+export const fittingFunctionDefinitions = [
+  {
+    id: 'None',
+    title: i18n.translate('xpack.lens.fittingFunctionsTitle.None', {
+      defaultMessage: 'none',
+    }),
+    description: i18n.translate('xpack.lens.fittingFunctionsDescription.None', {
+      defaultMessage: 'Do not fill gaps',
+    }),
+  },
+  {
+    id: 'Zero',
+    title: i18n.translate('xpack.lens.fittingFunctionsTitle.Zero', {
+      defaultMessage: 'zero',
+    }),
+    description: i18n.translate('xpack.lens.fittingFunctionsDescription.Zero', {
+      defaultMessage: 'Fill gaps with zeros',
+    }),
+  },
+  {
+    id: 'Linear',
+    title: i18n.translate('xpack.lens.fittingFunctionsTitle.Linear', {
+      defaultMessage: 'linear',
+    }),
```

Comment:
I think you swapped the meaning of my upper/lowercase comment and the result is that these are the opposite of what I was expecting. The I18n name should be capitalized for sure, and I also had a preference to have lowercase `id` fields.  As this is currently set up, it looks wrong:

<img width="386" alt="Screenshot 2020-06-30 18 14 02" src="https://user-images.githubusercontent.com/666475/86182637-580e1980-bafe-11ea-94c4-b995f2710ffe.png">

Reply:
I agree that id's should be lowercase, though I do think the visible values should be lowercase as well if you think about an input and it's label being more like sentence, you wouldn't capitalize "None" in a sentence.